assets/src/components/imageUpload/image_upload_24px.svg

```diff
@@ -0,0 +1,14 @@
+<svg width="24px" height="24px" viewBox="0 0 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
`@wordpress/components` has a few SVG-related classes (`SVG`, `Path`, `G`) that offer improved accessibility. They're used pretty regularly for block icons and other assets. Do you think we should use these here? 

https://github.com/WordPress/gutenberg/tree/master/packages/components/src/primitives/svg

Reply:
Seems like it may be a good idea. I wasn't able to get it working properly in my experiment with it, though. If you want to make an issue, I can spend more time investigating this for our icons when I have a chance.

Reply:
Done: https://github.com/Automattic/newspack-plugin/issues/49